<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SanjivaniK-LPT-234" failures="1" tests="1" name="RegistrationWithInvalidPhoneNumberField-9" time="13.388" errors="0" timestamp="2020-01-06T12:02:18 IST">
  <testcase classname="webAutomation.RegistrationTest.Register.Demo" name="validRegistrationTest" time="2.234">
    <failure type="java.lang.AssertionError" message="expected [https://accounts.lambdatest.com/email/verify] but found [https://accounts.lambdatest.com/register]">
      <![CDATA[java.lang.AssertionError: expected [https://accounts.lambdatest.com/email/verify] but found [https://accounts.lambdatest.com/register]
at webAutomation.RegistrationTest.Register.Demo.validRegistrationTest(Demo.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- validRegistrationTest -->
</testsuite> <!-- RegistrationWithInvalidPhoneNumberField-9 -->
